woocommerce/woocommerce-free-shipping-progress-bar-block
# Add initial e2e tests

Fixes #3 

## Note

At the moment, the plugin does not contain any e2e tests. This causes the risk that future developments may break existing functionality, but these breaks may remain undetected. By adding e2e tests, any break should be detected immediately.

## Testing instructions

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env start` to start the testing environment.
4. Run `npm run test:e2e` to run the e2e tests.
5. Ensure that all test cases pass.
